### PR TITLE
Add support for passing the model programmatically

### LINKED ISSUE #2: Expose programmatic way of loading a model
According to official documentation, it is possible to instantiate casbin model programmatically instead of loading it from file. See https://casbin.org/docs/en/model-storage#load-model-from-code

Would you be open to a PR that would implement support for the same in `fastify-casbin`?

### PR DESCRIPTION
**Pull request by kibertoad**
fixes #2

(this seems not to work, fixing now)

### CODE REVIEW

**Comment by kibertoad** [comment]
@simoneb Do you think it makes sense to narrow down possible type for enforcerParam? `casbin` types are not very helpful here, they define params as `any[]`

**Comment by kibertoad** [comment]
Having thought about it more, semver major approach seems more elegant and less complex.

File: test/casbin.test.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ const proxyquire = require('proxyquire')
 const sinon = require('sinon')
 
 const plugin = require('../')
+const { Model, FileAdapter } = require('casbin')
```
Comment (simoneb) [inline test/casbin.test.js:11]: please move this together with the external requires at the top

**Review by simoneb — APPROVED** [review]
LGTM. this will require a major semver bump as modelPath is replaced by model. the change makes sense to me anyway, modelPath was misleading

**Comment by simoneb** [comment]
> @simoneb Do you think it makes sense to narrow down possible type for enforcerParam? `casbin` types are not very helpful here, they define params as `any[]`

yeah they're defined like that because there's quite a bit of logic that they apply to the arguments provided to newEnforcer, see https://github.com/casbin/node-casbin/blob/master/src/enforcer.ts#L356.

nevertheless, I like the slightly more typed way we do it here, which gives some guidance about how to do it while still keeping lots of the flexibility

**Comment by kibertoad** [comment]
@simoneb Should I do the version bump or you'll do that during the release process?

**Comment by simoneb** [comment]
> @simoneb Should I do the version bump or you'll do that during the release process?

I'll bumb before release, thanks

File: test/casbin.test.js (hunk 1 above)
Comment (kibertoad) [reply test/casbin.test.js]: done

**Pull request merged**